spec/client_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "spec_helper"
+require "teamsnap"
+
+RSpec.describe TeamSnap::Client do
```

Comment:
@kyleries @semmons99 @jjmocko  what do you think of both styles of init like these spec out?